lib/LTP/utils.pm

```diff
@@ -152,7 +152,11 @@ sub export_ltp_env {
 
 # Set up basic shell environment for running LTP tests
 sub prepare_ltp_env {
-    assert_script_run('export LTPROOT=' . get_ltproot() . '; export LTP_COLORIZE_OUTPUT=n TMPDIR=/tmp PATH=$LTPROOT/testcases/bin:$PATH');
+    # $LTPROOT/testcases/realtime/func/sched_football/ is woraround for upstream bug
```

Comment:
woraround -> workaround

Reply:
Thank you, fixed (more precise info added).